app/forms/idv/api_image_upload_form.rb

```diff
@@ -79,8 +79,11 @@ def validate_form
     end
 
     def post_images_to_client
-      timer = JobHelpers::Timer.new
+      # user submit a request, set the requested_at timestamp
+      document_capture_session.requested_at = Time.zone.now
+      document_capture_session.save!
```

Comment:
persist it immediately in case we get racing condition in poller

Reply:
This timestamp is used at another place 
https://github.com/18F/identity-idp/blob/9a227356d5880354f7967d755c4a6f07e3df47e3/app/controllers/concerns/idv/verify_info_concern.rb#L23-L23

In this case, there is no racing concern? so no need to persist immediately.

Also, should we reset it after doc auth since it use used at this place also, maybe not.